core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -892,7 +894,22 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
       }
     }
 
-  def getTags(query: Option[String], limit: Option[Int] = None): Future[Seq[WorkspaceTag]] =
+  private def dumpLogs(): Unit = {
```

Comment:
This is a fake crash I created to simulate an error when querying the tags endpoint; I did this because I wasn't able to easily get logging to work in the test environment (it's stuck on the NOP logger despite my appeals)